tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -221,16 +223,29 @@ class MklDnnConvUtil {
       int filter_cols = static_cast<int>(filter_shape.dim_size(1));
       int in_depth = static_cast<int>(filter_shape.dim_size(2));
       int out_depth = static_cast<int>(filter_shape.dim_size(3));
-
-      // MKL-DNN always needs filter in OIHW format.
+      // MKL-DNN always needs filter in OIHW format for regular convolutions
```

Comment:
`filter` -> `filter shape (filter_dims)`
(Using just `filter` confused me with the actual filter tensor which is in TF layout.)